pkg/operator/staticpod/controller/missingstaticpod/missing_static_pod_controller.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 	"time"
 
+	configv1informers "github.com/openshift/client-go/config/informers/externalversions/config/v1"
```

Comment:
nit: we usually group imports, move down to line `19`

Reply:
Done